plenty_api/api.py

```diff
@@ -886,6 +888,15 @@ def plenty_api_get_property_selections(self, refine: dict = None):
 
         return selection_map
 
+    def plenty_api_get_property_selection_names(self, selection_id: int):
+        """
+        Get all Selection IDs and there names for specific selection Id.
```

Comment:
Hmm... Get all selection IDs for a specific selection ID?
+ s/there names/their names/
Also, please add the parameters and return part to the description to align with the other methods.